pkg/vpc/vpc.go

```diff
@@ -118,6 +118,9 @@ func UseFromCluster(provider api.ClusterProvider, stack *cfn.Stack, spec *api.Cl
 		outputs.ClusterSubnetsPublic: func(v string) error {
 			return ImportSubnetsFromList(provider, spec, api.SubnetTopologyPublic, strings.Split(v, ","))
 		},
+		outputs.ClusterSubnetsPublicLegacy: func(v string) error {
+			return ImportSubnetsFromList(provider, spec, api.SubnetTopologyPublic, strings.Split(v, ","))
```

Comment:
This seems like it's the same as the ClusterSubnetsPublic above, is this actually adding something?

Reply:
So it's meant to pick up one of them.
However, looking at this for the second time, I'd rather rework it a little bit. Let me take another  stab at it.

Reply:
I changed it so that it will only look for legacy key when non-legacy one is missing.